app/src/androidTest/java/org/mozilla/fenix/ui/TabbedBrowsingTest.kt

```diff
@@ -215,4 +216,53 @@ class TabbedBrowsingTest {
             }
         }
     }
+
+    @Test
+    fun closePrivateTabTest() {
+        var genericURLS = TestAssetHelper.getGenericAssets(mockWebServer)
+
+        homeScreen {
+        }.togglePrivateBrowsingMode()
+        genericURLS.forEachIndexed { index, element ->
+            navigationToolbar {
+            }.openNewTabAndEnterToBrowser(element.url) {
+                verifyPageContent(element.content)
+            }.openHomeScreen { }
+
```

Comment:
From `openHomeScreen { }` you can jump directly to the verification calls below without needing another `homeScreen` below since it transitions to that robot. E.g, https://github.com/mozilla-mobile/fenix/blob/029bbd37268bc6a1778730751f40d103baa9b2d3/app/src/androidTest/java/org/mozilla/fenix/ui/TopSitesTest.kt#L62

Reply:
missed that one. thanks